pom.xml

```diff
@@ -127,10 +127,11 @@
       <artifactId>jackson-dataformat-yaml</artifactId>
       <version>${jackson.version}</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.kotcrab.remark/remark -->
     <dependency>
-      <groupId>com.overzealous</groupId>
+      <groupId>com.kotcrab.remark</groupId>
```

Comment:
Yes, seems original artifact was removed from repo, so we should replace it with possible fork

Reply:
Ok I see